fluid/machine_translation.py

```diff
@@ -256,9 +256,15 @@ def train():
     optimizer = fluid.optimizer.Adam(learning_rate=args.learning_rate)
     optimizer.minimize(avg_cost)
 
+    fluid.memory_optimize(fluid.default_main_program())
+
+    # train_batch_generator = paddle.batch(
+    #     paddle.reader.shuffle(
+    #         paddle.dataset.wmt14.train(args.dict_size), buf_size=1000),
+    #     batch_size=args.batch_size)
+
     train_batch_generator = paddle.batch(
-        paddle.reader.shuffle(
-            paddle.dataset.wmt14.train(args.dict_size), buf_size=1000),
+            paddle.dataset.wmt14.train(args.dict_size),
```

Comment:
Why did we need to remove the shuffle? Or it just for testing?

Reply:
Yes, just for testing. We should fix the data order to get accurate result